src/custom/components/Web3Status/Web3StatusMod.tsx

```diff
@@ -139,43 +135,39 @@ function Sock() {
   )
 }
 
-/* function WrappedStatusIcon({ connector }: { connector: AbstractConnector | Connector }) {
-  return (
-    <IconWrapper size={16}>
-      <StatusIcon connector={connector} />
-    </IconWrapper>
-  )
-} */
+export function Web3StatusInner({ pendingCount }: { pendingCount: number }) {
+  const { account, connector, chainId, ENSName } = useWeb3React()
+  const connectionType = getConnection(connector).type
 
-export function Web3StatusInner({
-  pendingCount,
-  StatusIconComponent,
-  thereWasAProvider, //mod
```

Comment:
Don't we still need this mod for something?

Reply:
No, this is a part of previous eager connect to last wallet which is now changed